addons/core/translations/resources/en-us.yaml

```diff
@@ -706,7 +706,7 @@ target:
       help: The local proxy port on which to listen by default when a session is started on a client.
     sidebar:
       label: Session Recording
-      help: Record SSH sessions on this target and save them to an Amazon S3 bucket.
+      help: Record SSH sessions on this target and save them to a storage bucket.
```

Comment:
Good catch! I wonder if there's any other ones in the codebase?

I found this [one](https://github.com/hashicorp/boundary-ui/blob/c4e58fb780405eba2ee5f2e8f71268bcc0e24c40/addons/core/translations/resources/en-us.yaml#L450) but I don't think it's used?

Reply:
good catch! the help translation was def. not used anywhere.